### DISCUSSION TITLE
Where do I find the endpoint slack histogram?

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#3417] [General]
I'm running OpenROAD v2.0-8374-g5235ee26c, but I can't find the endpoint slack histogram. I thought it would be in the Windows menu...

https://github.com/The-OpenROAD-Project/OpenROAD/pull/3344

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/d979d815-5fee-4951-b4c4-dfb58c45e7a7)

**Comment by rovinski** [comment]
Do you have Qt5 charts installed? Your CMake log should have:
```
-- Charts widget is enabled
```
It requires a new optional dependency. On Ubuntu you should be able to do `apt install libqt5charts5-dev`.

**Comment by oharboe** [comment]
Very cool!

Some nits:

No legend on horizontal axis and I have to undock it and increase the size to see what's going on(no numbers visible when it is a tiny chartlet in the default position).

Also, I think the scripting window could leave space for the vertical column where the timing report is, instead of taking the full width of the window.

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/cbe9a37a-0eba-4cf6-8e5d-df51c58e2457)
